### PR TITLE
Update default regex colors for light/blue themes

### PR DESCRIPTION
**Pull request by dpoeschl**
Other theme colors handled in a VS PR.

**I will post screenshots soon.**

### CODE REVIEW

File: src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs [hunk 1]
```diff
@@ -1,7 +1,5 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-#define dark_theme
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs]: Wait, we were compiling with this all along?

Comment (Neme12) [reply src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs]: Tagging @CyrusNajmabadi

Comment (CyrusNajmabadi) [reply src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs]: Nice! :)

**Comment by CyrusNajmabadi** [comment]
Screenshots def welcome.

File: src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs [hunk 2]
```diff
@@ -378,19 +376,19 @@ private XmlDocCommentTextFormatDefinition()
...
-        private static readonly Color s_regexGroupingAndAlternationColor = Color.FromRgb(0x05, 0xc3, 0xba);
-        private static readonly Color s_characterClassColor = Color.FromRgb(0x00, 0x8a, 0xff);
-        private static readonly Color s_regexAnchorAndQuantifierColor = Color.FromRgb(0xd7, 0x45, 0x8c);
-        private static readonly Color s_regexCommentColor = Color.FromRgb(87, 166, 74);
+        private static readonly Color s_regexTextColor = Color.FromRgb(0xD6, 0x9D, 0x85);
+        private static readonly Color s_regexOtherEscapeColor = Color.FromRgb(0xFF, 0xD6, 0x8F);
+        private static readonly Color s_regexGroupingAndAlternationColor = Color.FromRgb(0x05, 0xC3, 0xBA);
+        private static readonly Color s_characterClassColor = Color.FromRgb(0x2E, 0xAB, 0xFE);
+        private static readonly Color s_regexAnchorAndQuantifierColor = Color.FromRgb(0xF9, 0x79, 0xAE);
+        private static readonly Color s_regexCommentColor = Color.FromRgb(0x4A, 0xBA, 0x39);
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs]: ok.  so textColor, otherEscape, and grouping/alternation did not change?   but characterclass/anchor did?  Did 'comment color' change?

**Comment by CyrusNajmabadi** [comment]
Note: kasey had a bunch of examples that were being used to ensure good contrast between groups.

**Comment by CyrusNajmabadi** [comment]
Things like:

```c#
        var x10 = new Regex(@"a\p{Pc}\u0011[\p{Lu}]\p{C}");
        var v1 = new Regex(@"$(\a\t\u0020)|[^\p{Lu}-a\w\s\(\)a-z-[m-p]]+?|(\b\G\z)(?<name>sub){0,5}?^");

        var v2 = new Regex(@"^((http|ftp|https|www)://)?([\w+?\.\w+])+([a-zA-Z0-9]\~\!\@\#\$_\.\:\;\'\,*)?$");
        var v3 = new Regex(@"(?<foo>\d+)");
        var v4 = new Regex(@"^[A-Za-z\n\ca\u00C1\u00C9\u00CD\u00D3\u00DA\u00E1\u00E9\u00ED\u00F3\u00FA]
                              [A-Za-z\u00C1\u00C9\u00CD\u00D3\u00DA\u00E1\u00E9\u00ED\u00F3\u00FA0-9@#%&\'-\s.\,]$");

        var v5 = new Regex(@"^((http|ftp|https|www)://)?([\w+?.\w+])+([a-zA-Z0-9~!\@#\$\%\^\&*()\-_=+\\/\?.\:\;\'\,]*)?$");

        var v6 = new Regex(@"\b(Kh?|Gh?|Qu?)[aeu](d['dt]?|t|zz|dhd)h?aff?[iy]\b(?#Ghaddafi)");
        var v7 = new Regex(@"\b(Kh?|Gh?|Qu?)[aeu]\1\2");
```

It would be good to see in practice, as a bunch of design was around how to make contrast work between things that commonly were next to each other.  Searching gitter for our old convos

**Comment by CyrusNajmabadi** [comment]
> note: dark comment color is: 87, 166, 74
i kept that
but otherwise, using your colors

image

File: src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs (hunk 1 above)
Comment (dpoeschl) [reply src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs]: Yeah, oops.

File: src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs [hunk 3]
```diff
@@ -378,19 +376,19 @@ private XmlDocCommentTextFormatDefinition()
...
         private static readonly Color s_regexTextColor = Color.FromRgb(0x80, 0x00, 0x00);
         private static readonly Color s_regexOtherEscapeColor = Color.FromRgb(0x9e, 0x5b, 0x71);
-        private static readonly Color s_regexGroupingAndAlternationColor = Color.FromRgb(0x05, 0xc3, 0xba);
-        private static readonly Color s_characterClassColor = Color.FromRgb(0x00, 0x73, 0xff);
-        private static readonly Color s_regexAnchorAndQuantifierColor = Color.FromRgb(0xff, 0x00, 0xc1);
-        private static readonly Color s_regexCommentColor = Color.FromRgb(0, 128, 0);
+        private static readonly Color s_regexGroupingAndAlternationColor = Color.FromRgb(0x29, 0x5F, 0x5B);
+        private static readonly Color s_characterClassColor = Color.FromRgb(0x05, 0x50, 0xC9);
+        private static readonly Color s_regexAnchorAndQuantifierColor = Color.FromRgb(0x9B, 0x20, 0x7C);
+        private static readonly Color s_regexCommentColor = Color.FromRgb(0x89, 0x42, 0x5C);
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs]: this surprises me.  what was wrong with the prevoius color?

File: src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs (hunk 2 above)
Comment (CyrusNajmabadi) [reply src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs]: i'm surprised that the comment color changed.  i thought we were using the actual Roslyn comment color.  so if that color was ok, this color should be ok too... right?

File: src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs (hunk 3 above)
Comment (CyrusNajmabadi) [reply src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs]: `0, 128, 0` is the standard color for comments.  So we should preserve that.  We only expose hte ability to change this if the user wants.

**Comment by dpoeschl** [comment]
Immediately known issues: Comment color and escaped character color in Blue/Light.

**Comment by CyrusNajmabadi** [comment]
quantifiers look very similar to anchors in teh blue themes.  intentional?

**Pull request closed without merging**